mupq.py

```diff
@@ -66,8 +66,11 @@ def run_make(self, target):
         makeflags.append(target)
         return subprocess.check_call(makeflags)
 
-    def get_binary_path(self, test_type, bin_type):
-        return f'bin/{self.path.replace("/", "_")}_{test_type}.{bin_type}'
+    def get_binary_path(self, test_type, bin_type=None):
+        if not bin_type:
```

Comment:
I think in python one should always check `bin_type is not None`, because there can be some weirdness with `None`.

Reply:
or `bin_type is None` of course in this case.

Reply:
changed. Thanks!